client/www/scripts/vendor/nv.d3.js

```diff
@@ -855,11 +855,12 @@ window.nv.tooltip.* also has various helper methods.
               }
 
               setTimeout(function() {
-
                   while (purging.length) {
                      var removeMe = purging.pop();
+                  if (removeMe.parentNode) {
```

Comment:
this is an issue with the nvd3 framework so will need to go through a pr on their repo